tests/phpunit/Mutant/MutantBuilder.php

```diff
@@ -37,17 +37,22 @@
 
 use Infection\Mutant\Mutant;
 use Infection\Mutation\Mutation;
+use InvalidArgumentException;
 use function Later\now;
 
 final class MutantBuilder
 {
     public static function build(
-        string $mutantFilePath,
-        Mutation $mutation,
-        string $mutatedCode,
-        string $diff,
-        string $prettyPrintedOriginalCode,
+        string $mutantFilePath = '/path/to/mutant',
+        ?Mutation $mutation = null,
+        string $mutatedCode = 'mutated code',
+        string $diff = 'diff',
+        string $prettyPrintedOriginalCode = '<?php $a = 1;',
     ): Mutant {
+        if ($mutation === null) {
+            throw new InvalidArgumentException('Mutation cannot be null');
+        }
```

Comment:
We can also make the mutation argument required, and reorder it to become first argument, but it will create a lot of irrelevant changes elsewhere.